elasticapm/contrib/aiohttp/middleware.py

```diff
@@ -31,11 +31,16 @@
 from aiohttp.web import middleware
 
 import elasticapm
-from elasticapm.conf import constants
 from elasticapm.contrib.aiohttp.utils import get_data_from_request, get_data_from_response
 from elasticapm.utils.disttracing import TraceParent
 
 
+class AioHttpTraceParent(TraceParent):
+    @classmethod
+    def merge_duplicate_headers(cls, headers, key):
+        return ",".join(headers.getall(key))
```

Comment:
Good catch. And validates your decision to make the dummy method as well! :)

Reply:
To be honest, looking at aiohttp led me to add that dummy method in the first place 😅 no crystal balls involved I'm afraid